src/ansys/fluent/core/services/datamodel_se.py

```diff
@@ -1480,6 +1480,23 @@ def _get_child_object_display_names(self) -> list[str]:
             )
         return child_object_display_names
 
+    def _get_id_name(self):
```

Comment:
@prmukherj This method is called `_get_id_name()` but it returns a list of names, which contradicts the method name. Even still, I don't know what the method name means.